### DISCUSSION TITLE
Month-by-month charts in multi-year reports

### DISCUSSION THREAD

**Discussion by tossha** [firefly-iii/firefly-iii#9454] [Firefly III] [answered]
Hi!
I'm making a tag (or category) report from start of 2022 to end of 2024. I really want to see a monthly chart of about 36 months, but it goups the data by year and I only see three large bars. Is it possible to force it to group by month instead of by year for mutli-year periods?

Thanks a lot!

![image](https://github.com/user-attachments/assets/1651dda8-83da-4c86-a1d0-fd9bfde59877)

**Comment by JC5** [comment, marked as answer]
I'm afraid it's not, but this is on my list to improve.

**Reply by tossha** [reply]
Thanks for a quick reply! Can't wait!
